src/data/faq.json

```diff
@@ -102,7 +226,7 @@
                             "<b> Questions and answers about the QR code:</b>",
                             "<a href='../faq/#check_in_qr_code' target='_self' rel='noopener noreferrer'> Who generates the QR code?</a></br><a href='../faq/#check_in_qr_code' target='_self' rel='noopener noreferrer'> Can the QR code also be set up and used for a private birthday party, providing this is legally possible?</a></br><a href='../faq/#check_in_qr_code' target='_self' rel='noopener noreferrer'>Where can the QR code be generated?</a></br><a href='../faq/#check_in_qr_code' target='_self' rel='noopener noreferrer'> Will there be a way to check in without a smartphone (dongle or similar)?</a></br><a href='../faq/#check_in_qr_code' target='_self' rel='noopener noreferrer'>How should I scan the QR code?</a></br><a href='../faq/#check_in_qr_code' target='_self' rel='noopener noreferrer'> What happens if I scan a QR code but do not have the CWA installed on my smartphone?</a>",

```

Comment:
The links can be reduced from ".../faq/#" to "#"

Reply:
indeed.